build.sbt

```diff
@@ -265,12 +268,7 @@ lazy val V = new {
 
 val sharedSettings = List(
   libraryDependencies ++= crossSetting(
-    scalaVersion.value,
-    if2 = List(
-      compilerPlugin(
-        "org.scalameta" % "semanticdb-scalac" % V.scalameta cross CrossVersion.full
-      )
-    )
```

Comment:
So do we have to include this like this, or can we do like I did up above and instead set `semanticdbEnabled := true` for everything? We are still able to set the desired `semanticdbVersion` in that way. If that's ok it sort of makes adding sourcegraph in here much easier since if not we'll need to add some extra settings since the sourcegraph upload will fail if `semanticdbEnabled` isn't set to true, which is isn't for us. Also, even if it was it will look for the semanticdb where `semanticdbTargetRoot` is set to, but the way we have this setup, we are always putting the semanticdb in the class directory, where Scalafix defaults to. This change (and the stuff up in `inThisBuild`) will instead let sbt handle the enabled, we set the correct version, and then set `semanticdbIncludeInJar := false`. Thoughts on this?

Reply:
crossSetting(scalaVersion.value) doesn't seem to do anything now.

Reply:
🤔 that's odd. I'm not sure. Let me dig in more.

Reply:
Ah yea true. I'll fix this but still can't get it to work. I'll need to dig in some more.

Reply:
Scratch this, this won't work this way since we need this for the properties files that are created in tests...